be/src/runtime/runtime_state.h

```diff
@@ -275,7 +275,7 @@ class RuntimeState {
 
     Status _build_global_dict(const GlobalDictLists& global_dict_list, vectorized::GlobalDictMaps* result);
 
-    static const int DEFAULT_CHUNK_SIZE = 2048;
+    static const int DEFAULT_CHUNK_SIZE = 4096;
```

Comment:
I think you should leave some comment here to tell people this can not be changed and why

Reply:
fixed